apps/src/pythonlab/pyodideWebWorker.ts

```diff
@@ -34,6 +34,15 @@ async function loadPyodideAndPackages() {
   });
   pyodide.setStdout(getStreamHandlerOptions('sysout'));
   pyodide.setStderr(getStreamHandlerOptions('syserr'));
+  // Freeze the module object and its properties
+  Object.freeze(pythonlabInputModule);
+  Object.defineProperty(pythonlabInputModule.getInput, 'constructor', {
```

Comment:
Nit: could throw a more descriptive error, but maybe keeping it abstruse is better. 
```
  Object.defineProperty(pythonlabInputModule.getInput, 'constructor', {
    get() {
      throw new TypeError(
        "Access to 'constructor' is restricted for security reasons."
      );
    },
  });
  ```

Reply:
yeah if anyone is trying to do this they are up to no good, so I think it's fine to leave as-is